src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div class="product" data-qa="card">
+        <img
+          class="product__photo"
+          src="images/imac.jpeg"
+          alt="imac photo"
+        >
+
+        <p class="product__text-name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+
+        <p class="product__text-code">
+          Код товара: 195434
+        </p>
+
+        <div class="stars stars--4">
+          <span class="stars__star stars__star--1"></span>
+          <span class="stars__star stars__star--2"></span>
+          <span class="stars__star stars__star--3"></span>
+          <span class="stars__star stars__star--4"></span>
+          <span class="stars__star stars__star--5"></span>
+
+          <p class="stars__review">Отзывов: 5</p>
```

Comment:
reviews shouldn't be part of stars block. create a wrapper around stars and reviews and give it display: flex and justify-content: space-between